### DISCUSSION TITLE
icons to text on Nav bar

### DISCUSSION THREAD

**Discussion by gyaltnnh** [Aris-t2/CustomCSSforFx#536] [Q&A]
Many versions of FF ago, one could change the icons on the navigation bar to text.  I was able to keep that option until version 108.0.1.  Is there a new css file to do so?  I am not css literate so i have used Aris-t2's work in the past.  thanks!

**Comment by Aris-t2** [comment]
You are probably referring to the option `toolbar_mode_text.css`.

**Reply by gyaltnnh** [reply]
At your suggestion, I tried that, and it made all of the icons disappear, , but no text appeared.

**Reply by Aris-t2** [reply]
Default setup of CustomCSSforFx 4.2.8 release:

![image](https://user-images.githubusercontent.com/6013569/209845168-4c8c1dee-efef-413b-88d5-c539774d4586.png)

`userChrome.css` > enabled `toolbar_mode_text.css`

![image](https://user-images.githubusercontent.com/6013569/209845225-e7cfe251-4346-439c-957f-0ca4943ec03f.png)

**Comment by gyaltnnh** [comment]
Obviously, I am doing something wrong. I copied the highlighted file, pasted it into userchrome, this is the result
[css.pdf](https://github.com/Aris-t2/CustomCSSforFx/files/10327668/css.pdf)

_Edit (added screenshot to the post)_
![image](https://user-images.githubusercontent.com/6013569/212030138-1b46b746-b654-4d68-89bc-236c3c84fd0f.png)

**Reply by Aris-t2** [reply]
Use the [full release](https://github.com/Aris-t2/CustomCSSforFx/releases) and enable the feature inside `userChrome.css` file.

**Reply by gyaltnnh** [reply]
What do you mean by "enable the feature inside userChrome.css file."?

**Reply by Aris-t2** [reply]
https://github.com/Aris-t2/CustomCSSforFx#how-to-use-custom-user-styles

**Reply by gyaltnnh** [reply]
Is it possible that there is a conflict between this and tabs_below_navigation_toolbar.css?

**Reply by Aris-t2** [reply]
I looked again at your screenshot. There is a possibility the conflict is caused by your theme. You are using Windows Classic theme, that might use white text color in that combination causing toolbar button text to stay invisible.

Test a different theme.

**Comment by Aris-t2** [comment]
There must be so other conflict on your end. I just tested Win 7 + Windows Classic theme in a VM only enabling tab below navigation toolbar and toolbar text mode.

This is, what I have used in that test. Replace your "chrome" folder content with this one and make sure to use default browser theme:
[chrome.zip](https://github.com/Aris-t2/CustomCSSforFx/files/10400079/chrome.zip)

![image](https://user-images.githubusercontent.com/6013569/212029907-902637b4-0d49-4acd-bf95-a8969d6d9aac.png)

**Comment by gyaltnnh** [comment]
Sir,
You are correct.  I used the chrome file above, but can't find a "default browser theme" choice, so I used one called Dark 1.2. With that theme, the nav bar shows up with white letters.  Not a fan of dark themes, but it will work.  BTW, I would like to contribute to your work, but when I follow the link to Paypal, a person's name shows up as the payee, and I'm not sure if I'm in the right place.

**Reply by Aris-t2** [reply]
The Paypal link is correct, it me.
Thanks.
